newIDE/app/src/EffectsList/index.js

```diff
@@ -486,213 +715,188 @@ export default function EffectsList(props: Props) {
+                                }}
+                                canDrag={() => true}
+                                canDrop={() => true}
+                                drop={() => {
+                                  moveEffect(effect);
+                                }}
+                              >
+                                {({
+                                  connectDragSource,
+                                  connectDropTarget,
+                                  isOver,
+                                  canDrop,
+                                }) =>
+                                  connectDropTarget(
+                                    <div
+                                      key={effect.ptr}
+                                      style={styles.rowContainer}
+                                    >
+                                      {isOver && (
+                                        <DropIndicator canDrop={canDrop} />
+                                      )}
+                                      <Effect
+                                        ref={effectRef}
+                                        layerRenderingType={'3d'}
+                                        target={target}
+                                        project={project}
+                                        resourceManagementProps={
+                                          props.resourceManagementProps
+                                        }
+                                        effectsContainer={effectsContainer}
+                                        effect={effect}
+                                        removeEffect={removeEffect}
+                                        copyEffect={copyEffect}
+                                        pasteEffectsBefore={pasteEffectsBefore}
+                                        chooseEffectType={chooseEffectType}
+                                        allEffectMetadata={all3DEffectMetadata}
+                                        onEffectsUpdated={onEffectsUpdated}
+                                        onEffectsRenamed={onEffectsRenamed}
+                                        nameErrors={nameErrors}
+                                        setNameErrors={setNameErrors}
+                                        connectDragSource={connectDragSource}
+                                      />
+                                    </div>
+                                  )
+                                }
+                              </DragSourceAndDropTarget3D>
+                            ) : null;
+                          }
+                        )}
+                      </Column>
+                    </Line>
                   </Column>
-                </Line>
+                )}
+                {props.layerRenderingType !== '3d' && effect2DCount > 0 && (
+                  <Column noMargin expand>
+                    {props.layerRenderingType !== '2d' && (
```

Comment:
I think it should be '3d' here no?

Reply:
The 1st condition is: "is there any 2D effect" --> show the section
The 2nd condition is: "is there not only 2D effect" --> show the title

Reply:
It struck my eyes because it's not symmetric to the 3D effects section rendering:
https://github.com/4ian/GDevelop/pull/5484/files#diff-f0003e608f9793a9a7c71122c721f1560f727374dc242ab276c8277e2af20392R718-R720

Reply:
Indeed, I'll fix this one